src/Microsoft.SymbolStore/SymbolStores/HttpSymbolStore.cs

```diff
@@ -75,6 +75,14 @@ public void ResetClientFailure()
         protected override async Task<SymbolStoreFile> GetFileInner(SymbolStoreKey key, CancellationToken token)
         {
             Uri uri = GetRequestUri(key.Index);
+
+            if (key.ChecksumAlgorithmName != null && key.ChecksumAsHex != null)
```

Comment:
Sorry if this is obvious, but I think we need validation support before we complete this PR. We could either build it directly into this library, or provide a call back if we want to avoid possible version conflicts with the portable PDB reader assembly.

Reply:
@gregg-miskelly , I don't understand what you mean by "validation support".

Reply:
This code should verify that the downloaded PDB has a hash that matches one of the hashes in the hash collections.

Reply:
I wondered about whether or not the validation support belonged in here. Thanks for the clarification.